src/patternfly/components/Label/label.scss

```diff
@@ -247,6 +252,18 @@
     }
   }
 
+  &.pf-m-editable {
+    &.pf-m-editable-active {
```

Comment:
With the class name having editable in it, I wonder if these need to be nested or if we could style the classes independently.